lib/logger.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:html';
+
+class Logger {
+  String file;
+
+  Logger(this.file);
+
+  void log(String message) {
+    var date = DateTime.now();
+    window.console.log("[${this.file} $date] $message");
```

Comment:
Consider refactoring the string generation int a separate method given that all of these do the same thing

Reply:
Ah yes, fixed!